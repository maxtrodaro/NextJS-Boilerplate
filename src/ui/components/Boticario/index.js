import * as S from './styles'

const Boticario = () => (
  <S.Wrapper>
    <h1>Boticario</h1>
  </S.Wrapper>
)

export default Boticario
