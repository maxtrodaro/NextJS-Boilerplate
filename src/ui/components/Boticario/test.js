import { render, screen } from '@testing-library/react'

import Boticario from '.'

describe('Boticario test component', () => {
  it('should render the heading', () => {
    const { container } = render(<Boticario />)

    expect(screen.getByRole('heading', { name: /Boticario/i })).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
