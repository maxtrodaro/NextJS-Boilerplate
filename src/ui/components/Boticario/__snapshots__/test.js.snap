// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Boticario test component should render the heading 1`] = `
<main
  class=""
>
  <h1>
    Boticario
  </h1>
</main>
`;
